tools/prepare_job.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+"""This script is designed to generate a Job-file in yaml, which will
+be used with our RevPi devices in order to flash and perform tests with
+LAVA automatically, thus saving time finding errors in the images.
+"""
+
+import argparse
+import os
+import sys
+import yaml
+
+NEW_JOB_NAME = "job_flash_test.yaml"
+REVPI_DEFAULT_DEVICE = "RevPi_Core"
+PATH_DEPLOYMENT_FLASH = "./automated/deployment/"
+PATH_TESTSUITES = "./automated/testsuites/"
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    epilog="Default behaviour: blablabla...",
+)
+
+parser.add_argument(
+    "-d",
+    "--device_type",
+    dest="device_type",
+    nargs="?",
+    default=REVPI_DEFAULT_DEVICE,
+    help="device type ...",
+)
+
+parser.add_argument(
+    "-u",
+    "--url",
+    dest="url",
+    required=True,
+    help="url ...",
+)
+
+parser.add_argument(
+    "-f",
+    "--flash",
+    dest="flash",
+    required=True,
+    help="flash ...",
```

Comment:
Maybe rename this to `-i`, `--image`? Ideally it should be clear what this flag does just from the name

Reply:
By -f I mean the name of the file used for flashing.
As there is currently only 1, this will be used as default.

For the image I used -u, because of the "url"... maybe here it makes more sense to use -i...

Reply:
`-u` for the image URL is fine by me, I usually assume `-f` to be `--force` though.